Blog for Running KRM functions from velero plugins

chaitanyab2311 opened the pull request.

chaitanyab2311 committed:
added blog post outline for running krm functions from velero plugins

chaitanyab2311 committed:
updated blog post for running krm functions from velero plugins

chaitanyab2311 committed:
Modified blog content for review and added flow diagram

chaitanyab2311 committed:
Added youtube video link for blog post

jwmatthews commented:
@chaitanyab2311 I see this is still in a draft state, do you want to move this to be ready to be merged?

chaitanyab2311 commented:
Hello John,
Yes the content and all parts are done and checked. I added a additional
part for adding a image so that it is not visible in ReadME and as there is
a static content it should come up when it's published however in preview
on github p that does not show up. That particular part I could not test as
it was not experimented on before by anyone so I am hoping it shows up
after publishing as I added the path according to the Hugo documentation.

Thanks and Regards,
Chaitanya

On Sat, Sep 24, 2022 at 5:24 AM John Matthews ***@***.***>
wrote:

> @chaitanyab2311 <https://github.com/chaitanyab2311> I see this is still
> in a draft state, do you want to move this to be ready to be merged?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/konveyor/eng-kbase/pull/32#issuecomment-1256945956>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVHIIBHP5ADAAP36PQVOQ2LV73QITANCNFSM555SK45A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
